internal/deploy/olm.go

```
@@ -0,0 +1,213 @@
+		return result, fmt.Errorf("timed out waiting for Deployment %q to be created in namespace %q",
+			cfg.DeploymentName, cfg.Namespace)
+	}
+
+	operatorDeployment, err := deployment.Pull(apiClient, cfg.DeploymentName, cfg.Namespace)
+	if err != nil {
+		return result, fmt.Errorf("failed to pull Deployment %q in namespace %q: %w",
+			cfg.DeploymentName, cfg.Namespace, err)
+	}
+
+	if !operatorDeployment.IsReady(deploymentReadyTimeout) {
+		return result, fmt.Errorf("Deployment %q in namespace %q did not become ready within %s",
```

golangci-lint (2.13.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)